apps/web/src/pages/studio-onboarding/index.tsx

```diff
@@ -0,0 +1,161 @@
+import { css } from '@novu/novui/css';
+import { vstack } from '@novu/novui/patterns';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { text, title } from '@novu/novui/recipes';
+import { styled } from '@novu/novui/jsx';
+import { CodeSnippet } from '../get-started/components/CodeSnippet';
+import { Timeline } from '../get-started/components/timeline/Timeline';
+import { useEffect, useMemo, useState } from 'react';
+import { Input } from '@novu/design-system';
+import { IconCheck } from '@novu/novui/icons';
+import { api, getApiKeys, ROUTES, useEnvController } from '@novu/shared-web';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { useNavigate } from 'react-router-dom';
+import { When } from '../../components/utils/When';
+
+const Heading = styled('h1', title);
+const Text = styled('p', text);
+
+export const StudioOnboarding = () => {
+  const [active, setActive] = useState(0);
+  const { data: apiKeys = [] } = useQuery<{ key: string }[]>(['getApiKeys'], getApiKeys);
+  const { environment } = useEnvController();
+  const key = useMemo(() => apiKeys[0]?.key, [apiKeys]);
+  const navigate = useNavigate();
+  const [url, setUrl] = useState('http://localhost:9999/api/echo');
+  const [error, setError] = useState('');
+  const { mutate: sync, isLoading: isSyncing } = useMutation(
+    (data: { chimeraUrl: string }) => api.post('/v1/echo/sync', data),
```

Comment:
same as below, new naming 🙃, its already merged to next

```suggestion
    (data: { bridgeUrl: string }) => api.post('/v1/echo/sync', data),
```